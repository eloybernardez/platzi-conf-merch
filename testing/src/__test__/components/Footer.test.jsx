import React from "react";
import { render, screen } from "@testing-library/react";
import Footer from "../../components/Footer";
import { expect, it, describe } from "vitest";

describe("<Footer/>", () => {
  it("Render del componente Footer", () => {
    render(<Footer />);
    expect(screen).toBeDefined();
  });

  it("Render del tÃ­tulo", () => {
    render(<Footer />);
    const title = screen.getByText("Platzi Store");
    expect(title).toBeDefined();
  });
});

describe("Footer Snapshot", () => {
  it("Comprobar la UI del componente Footer", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment(<Footer />)).toMatchSnapshot();
  });
});
