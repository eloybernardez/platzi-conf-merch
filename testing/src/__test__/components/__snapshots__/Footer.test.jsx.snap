// Vitest Snapshot v1

exports[`Footer Snapshot > Comprobar la UI del componente Footer 1`] = `
<DocumentFragment>
  <div
    class="Footer"
  >
    <p
      class="Footer-title"
    >
      Platzi Store
    </p>
    <p
      class="Footer-copy"
    >
      Todos los Derechos Reservados
    </p>
  </div>
</DocumentFragment>
`;
